netbout-web/src/main/java/com/netbout/email/EmAlias.java

```diff
@@ -119,6 +125,48 @@ public void email(final String email, final String urn, final Bout bout)
         new BoutInviteMail(this.postman).send(email, urn, bout);
     }
 
+    /**
+     * {@inheritDoc}
+     * @todo #738 We need to improve the format and content of the
+     *  verification email. Currently, it only asks the user to verify
+     *  the new email by using the provided verification link. We may add
+     *  some header, footer, note like "Ignore this email if you did not
+     *  change your email address", etc.
+     */
+    @Override
+    public void email(final String email, final String link)
+        throws IOException {
+        this.email(email);
+        try {
+            this.postman.send(
+                new Envelope.MIME()
+                    // @checkstyle IndentationCheck (12 lines)
```

Comment:
@caarlos0 I found it difficult to format correctly, maybe it has something to do with [this] checkstyle issue. However, I moved subject `with` clause one level up (before the recipient); it seems that checkstyle is happy now. :)